dialdbg/src/strings.rs

```diff
@@ -0,0 +1,21 @@
+// strings.rs contains substrings of dial log messages. These constants are admittedly the most
+// fragile piece of dialdbg's code. If dialdbg is not behaving as expected, ensure these constants
```

Comment:
(opt) can we make this less fragile by having the actual error messages point to these const strings as well? Would definitely buy "yes, but in a different PR" as a response.

Reply:
Yep! Added. @npmenard requested the same thing. I moved the strings to `rpc/log_prefixes` so we don't have a circular dependency.